release.Dockerfile

```diff
@@ -12,10 +12,8 @@ ENV UVICORN_PORT=6900
 
 # Copying script for starting argilla server by checking the elasticsearch status
 COPY scripts/start_argilla_server.sh /
-COPY scripts/wait-for-it.sh /
 
 RUN chmod +x /start_argilla_server.sh \
- && chmod +x /wait-for-it.sh \
  && for wheel in /packages/*.whl; do pip install "$wheel"[server]; done
 
 # Executing wait-for-it script followed by starting the argilla server
```

Comment:
Please update the comments

Reply:
Comments upgraded.

I prefer to keep the script wrapping all things related to launching the server, even if, for now, we execute one single command.